src/template.js

```diff
@@ -36,6 +36,41 @@ function formatValueToList(value) {
   }
 }
 
+function transformObjectData(objectData) {
+  if (getType(objectData) !== 'array') {
+    return objectData;
+  }
+
+  return objectData.map((od) => {
+    if (getType(od) !== 'object') {
+      return od;
+    }
+
+    // The API expects price to be a number so delete it entirely if empty.
+    if (getType(od.price) === 'string' && !od.price) {
+      od.price = undefined;
+    }
+
+    // The API expects discount to be a number so delete it entirely if empty.
+    if (getType(od.discount) === 'string' && !od.discount) {
```

Comment:
I'm thinking the same here, checking for empty string rather than omitting a (potentially) valid `0`.

Reply:
As above.